gulpfile.js

```diff
@@ -163,7 +163,7 @@ function getRunDebugAppCommand(arch) {
 
     case 'win32':
     case 'win64':
-        return path.join(DEBUG_DIR, pkg.name, arch, pkg.name + '.exe');
+            return path.join(DEBUG_DIR, pkg.name, arch, pkg.name + '.exe --remote-debugging-port=9222');
```

Comment:
This must go here? If you think is useful, this must be done in other PR to be discussed.

Reply:
Sorry, my mistake. It's to debug in VS Code...